articles/team-fit.md

```diff
@@ -0,0 +1,35 @@
+---
+authorName: Marko Simic
+authorGithubUsername: masizuehlke
+issue: 90
+title: Team Fit
```

Comment:
Proposals:
- harmonic collaboration with team fit
- 8 characteristics of team fit
- the perfect team fit
-

Reply:
Thank you for you feedback. 
Thinking about your proposals, thought took me do this direction:

- Yourself in world of measurable
- Two sides of complete professional 
- Moving as one
- Can we swim together?
- Are we good for you?
....(and still thinking :-))